guides/micronaut-turbo-native/src/main/resources/views/files.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE HTML>
+<html lang="en" th:replace="~{layoutFile :: layout(~{::section})}" xmlns:th="http://www.thymeleaf.org">
+<section class="grid">
+    <h1 class="page-title">Handling Files</h1>
+
+    <p>
+        Here's a link to an image. Turbo doesn't handle non-html links, so it will ignore this link. By default, it will open in the browser. We've added special handling in the app for files on the same domain to open in-app instead.
```

Comment:
```suggestion
        Here's a link to an image. Turbo doesn't handle non-html links, so it will ignore this link. By default, the image will open in the browser. We've added special handling in the app for files on the same domain to open in-app instead.
```